python/ray/scripts/scripts.py

```diff
@@ -386,7 +387,22 @@ def debug(address):
     required=False,
     default="{}",
     type=str,
-    help="a JSON serialized dictionary mapping resource name to resource quantity",
+    help="A JSON serialized dictionary mapping resource name to resource quantity."
+    + (
+        r"""
+
+Windows command prompt users must ensure to double quote command line arguments; since
+JSON requires the use of double quotes these must be escaped as well, e.g.
+
+    ray start --head --resources="{\"special_hardware\":1, \"custom_label\":1}"
```

Comment:
We can do `ray start --head --resources='{"special_hardware": 1}'` so we don't need to escape.

Reply:
Does this work for Windows as well? The example I put here is taken directly from an issue where a user tried to input bad JSON similar to this (but missing the escape sequences).

Reply:
Indeed the suggested single-quote does not work. Windows is weird.
```
>ray start --head --resources='{"special_hardware": 1}'
Usage: ray start [OPTIONS]
Try 'ray start --help' for help.

Error: Got unexpected extra argument (1}')
```

Reply:
Thanks for checking!